src/index.md

```diff
@@ -44,11 +44,13 @@ The ecosystem consist of various components:
 - [How to create a new project](./manage/create-project/)
 - [Home page and dashboard](./manage/dashboard/)
 - [How to share, transfer or delete a project](./manage/project-advanced/)
+- [How to use multiple servers in the Mergin Maps plugin for QGIS](./manage/plugin-multi-server-use/)
```

Comment:
This could be renamed to match the title of the page (How to Use Mergin Maps plugin with multiple servers)